### ISSUE TITLE
Smarty4 issue with template file

### ISSUE THREAD

**Issue by backofficeteam** [lcdservices/biz.lcdservices.movecontrib#24]
CiviCRM 5.75.3, MoveContribution 2.5, Smarty4.

Clicking on the move contribution link on a contribution yields this error:

  "message" => "Unable to load template 'file:CRM/LCD/MoveContrib/Form/MoveContrib.tpl' in 'file:CRM/Form/default.tpl'"
  "code" => null
  "exception" => SmartyException {#13258
    #message: "Unable to load template 'file:CRM/LCD/MoveContrib/Form/MoveContrib.tpl' in 'file:CRM/Form/default.tpl'"

**Comment by eileenmcnaughton** [comment]
I just checked this on our production config - and it is working on our combo - 5.77, Smarty5 & 2.12 of this extension - did you actually have the old 2.5 version of this extension - or was that a typo?

**Comment by lcdservices** [comment]
+1 to @eileenmcnaughton comment. I'm testing on 5.76.1 with MoveContribution 5.12, and tested with both Smarty 4 and 5, and it worked fine. I think you just need to upgrade the extension.

**Comment by backofficeteam** [comment]
No, I meant 2.5.  Interesting that the UI in Civi doesn't show an update is available: 
![image](https://github.com/user-attachments/assets/4efe7abc-21bf-4c6f-a19e-5adedf8d339d)

Thank you, we will upgrade.
